package.json

```diff
@@ -46,11 +46,13 @@
   },
 
   "scripts": {
-	"test": "mocha tests.coffee --compilers coffee:coffee-script/register"
+    "test": "mocha test/dice_test.coffee --compilers coffee:coffee-script/register"
```

Comment:
> Since the test passes you can merge this PR and then move tests.coffee in and do that part?

No :(

> The intention was to change it to test/* or tests/* when there are more tests. Is that okay?

Sure, but might as well do it now, and move the esisting `tests.coffee` into `test/` so it also runs